app/policies/ebook_operation.rb

```diff
@@ -20,8 +20,24 @@ def unrestricted?
     end
 
     def licensed_for?(entitlement)
-      authority
-        .licenses_for(actor, ebook)
-        .any? { |license| license.allows?(entitlement) }
+      licenses = authority.licenses_for(actor, ebook)
+
+      return true if licenses
+                       .where(licensee_type: "Greensub::Individual")
+                       .any? { |license| license.allows?(entitlement) } && actor.individual.present?
```

Comment:
&& actor.individual.present? is redundant since the actor must be an individual in order to have an individual license.